lib/ood_core/job/adapters/fujitsu_tcs.rb

```diff
@@ -52,9 +59,17 @@ class Fujitsu_TCS_TimeoutError < Error; end
 
           # @param bin [#to_s] path to Fujitsu TCS installation binaries
           # @param bin_overrides [#to_h] a hash of bin ovverides to be used in job
-          def initialize(bin: nil, bin_overrides: {})
+          # @param working_dir [] Working directory for submitting a batch script
+          def initialize(bin: nil, bin_overrides: {}, working_dir: nil)
             @bin           = Pathname.new(bin.to_s)
             @bin_overrides = bin_overrides
+            if working_dir == nil
+              @working_dir = Dir.pwd
+            elsif working_dir == "HOME"
+              @working_dir   = Dir.home
+            else
+              raise(Error, "Unknown working_dir")
```

Comment:
```suggestion
              raise(StandardError, "Unknown working_dir")
```

Should you ever hit this line - you'll get the error which may be ambiguous. 
```
irb(main):001:0> raise(Error, 'my message')
(irb):1:in `<main>': uninitialized constant Error (NameError)
Did you mean?  IOError
               Errno
        from /home/jeff/apps/ruby/3.0.6/lib/ruby/gems/3.0.0/gems/irb-1.3.5/exe/irb:11:in `<top (required)>'
        from /home/jeff/apps/ruby/3.0.6/bin/irb:23:in `load'
        from /home/jeff/apps/ruby/3.0.6/bin/irb:23:in `<main>'
```